ccd-definition/CaseField/CaseFieldGAspec.json

```diff
@@ -237,5 +237,12 @@
     "Label": " ",
     "FieldType": "GAUserDetailsGAspec",
     "SecurityClassification": "Public"
+  },
+  {
```

Comment:
we do not need this field and its corresponding authorizations.

Reply:
After talking to Smita - I deleted these values as they were not needed. Tested again works approx.